NAS2D/Resources/Music.h

```diff
@@ -39,7 +39,7 @@ class Music
 
 protected:
 	friend class MixerSDL;
-	void* music() const;
+	Mix_Music* music() const;
```

Comment:
This pollutes the interface with low-level implementation details which I'd hoped to avoid. Not going to nitpick it but that was the justification here for use of void pointer.

Reply:
I know. I'm not sure of a clean way to allow needed access, while keeping `Music` separate from `MixerSDL`.

That's why it's a `protected` method with `friend` only access. The only place that can use it, `MixerSDL`, will already be importing the SDL-mixer headers. The type is just forward declared for this file, to prevent header pollution affecting other users of the `Music` class.

Granted, it does create the small bit of pollution that there is an incomplete global type named `Mix_Music`, that will be defined somewhere.